### ISSUE TITLE
function svgResource() : Size of svg will be unspecified if the SVG has a ViewBox but does not cointain width and height.

### ISSUE THREAD

**Issue by Luxusproblem** [JetBrains/compose-jb#491]
Hello
First:  Thank you for this lovely Project. 

I found aissue you might be interested in: 

This code
````kotlin
Image(painter = svgResource("icon/test.svg"),"icon")

````
with those SVG attributes
````xml

<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"> 
...
</svg>
````

will throw this exception:

````
" java.lang.IllegalStateException: Size is unspecified"
````

If I manually add a width  and a height to the svg, it works. 
````xml

<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" width="32" height="32">  
...
</svg>
````
Is it possible to have a svg fallback to "viewbox" to derive a size if there is no width or height specified ?
Or a more Accurate exception, because this was really hard to figure out.  
Like 
````
Resource is missing the width or height attribute.
````

**Comment by Vahalaru** [comment]
Oh my gosh. Thank you! I was stuck for hours on that error.

**Comment by c5inco** [comment]
Yep just ran into this. The fact that you can't try/catch `svgResource` also makes this very hard to detect when trying to load 1000s of SVGs in a `LazyColumn`.

**Comment by c5inco** [comment]
Related, I now have an SVG that doesn't throw an exception when manually adding the `width` and `height`, but now just renders an empty image. This is the SVG in question: https://gist.github.com/c5inco/069cb43819f77725f08fa4a5e9adf647

Can others repro this?

**Comment by igordmn** [comment]
Should be fixed in 0.5.0-build224

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6416">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
